### PR TITLE
Issue #1870 - disable implicit bean archive scanning

### LINKED ISSUE #1870: A way to disable implicit bean archive in CDI 20
In CDI spec, it requires the implementation to provide a way to disable implicit bean archive scanning. In CDI 1.2, we have the following configuration in server.xml.

<cdi12 enableImplicitBeanArchives="false" />

In the current CDI 2.0, we have  the following configuration.
<cdi enableImplicitBeanArchives="false" />

After further discussion, we have agreed it is best to stick with the previous config so that no impact on migration.
Decision is to stick with cdi12 and update the element name and description to indicate that it is applicable to cdi-1.2 or newer features.

<cdi12 enableImplicitBeanArchives="false" />

Need to change the current codebase to reflect the newly agreed configuration.

### PR DESCRIPTION
**Pull request by tevans78**
#build 
fixes #1870 

This is the same as PR #2082 but with the config name shortened slightly

### CODE REVIEW

File: dev/com.ibm.ws.cdi.weld/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -13,8 +13,8 @@
 #NLS_ENCODING=UNICODE
 #NLS_MESSAGEFORMAT_NONE
 #
-cdi12=Contexts And Dependency Injection (CDI) V1.2
-cdi12.desc=Defines the behavior of the Contexts and Dependency Injection (CDI) v1.2.
+cdi12=Contexts And Dependency Injection V1.2 or newer
+cdi12.desc=Defines the behavior of the Contexts and Dependency Injection (CDI) v1.2 or newer.
```
Comment (tevans78) [inline dev/com.ibm.ws.cdi.weld/resources/OSGI-INF/l10n/metatype.properties]: @bcpswope please review these two lines

File: dev/com.ibm.ws.cdi.1.2.weld/bnd.bnd [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #*******************************************************************************
-# Copyright (c) 2017 IBM Corporation and others.
+# Copyright (c) 2017,2018 IBM Corporation and others.
 # All rights reserved. This program and the accompanying materials
```
Comment (bcpswope) [inline dev/com.ibm.ws.cdi.1.2.weld/bnd.bnd]: Needs an additional space between 2017,2018. (There may be more cases.)
2017, 2018

File: dev/com.ibm.ws.cdi.weld/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (bcpswope) [reply dev/com.ibm.ws.cdi.weld/resources/OSGI-INF/l10n/metatype.properties]: sorry--labeled but did not respond. Looks good.

Comment (bcpswope) [reply dev/com.ibm.ws.cdi.weld/resources/OSGI-INF/l10n/metatype.properties]: @tevans78 these look good, I only mentioned a change in the copyright statement.

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
Turns out that the feature title has to be fully Capitalized and not Title Case as the error message suggests.
#build

**Comment by tevans78** [comment]
Finally found a string that the feature checker is happy with. Ready to merge when integration is unlocked.

**Comment by tevans78** [comment]
Note that this is NOT needed for 18.0.0.1

**Comment by tevans78** [comment]
Rebased on integration so rebuilding
#build

**Comment by tevans78** [comment]
#build

**Pull request merged**